src/graph_world/nodeclassification/generator_wrappers.py

```diff
@@ -44,6 +45,13 @@ def __init__(self, param_sampler_specs, marginal=False,
     self._AddSamplerFn('power_exponent', self._SampleUniformFloat)
     self._AddSamplerFn('min_deg', self._SampleUniformInteger)
 
+    if use_generated_lfr_communities:
+      self._AddSamplerFn('mixing_param', self._SampleUniformFloat)
```

Comment:
`mixing_param` does not affect LFR community generation, correct? If so, we should not at a sampler fn for it in the SBM generator.